cli/src/pcluster/constants.py

```diff
@@ -33,6 +33,12 @@
     "ubuntu2004": {"user": "ubuntu", "root-device": "/dev/sda1"},
 }
 
+# Describe the list of requirements to be satisfied by the Pcluster AWS Batch CLI to manage the cluster.
+# It must be in the form <package-name><comparison-operator><version>
+# It can contain multiple items separated by a colon.
+# i.e. aws-parallelcluster-awsbatch-cli>=2.0.0,aws-parallelcluster-awsbatch-cli<3.0.0
+AWSBATCH_CLI_REQUIREMENTS = "aws-parallelcluster-awsbatch-cli<2.0.0,aws-parallelcluster-awsbatch-cli>=1.1.0"
```

Comment:
why 1.1.0?

Reply:
I changed it for testing, fixed, good catch!